atlp-rwanda/atlp-pulse-mobile
# Setup validation

### What does this PR do?

This PR introduce a way to validate forms

### Description of Task to be completed?

Introduced a new dependency `formik` a library used to validate forms using `yup`.

### How should this be manually tested?

- Run `npm install` to install new dependencies.
- `npm start` and open the app in expo go app
- Check validations on the first page.

### Developer Notes

Validation testing page will be removed after this PR is approved.

Checkout the documentation on `formik` with react-native [here](https://formik.org/docs/guides/react-native)

### Screenshots

N/A
